data/gen_graphs.py

```diff
@@ -0,0 +1,88 @@
+import random
+import networkx as nx
+import subprocess
+
+
+def gen_erdos_renyi_graph_single_component(n, m):
+    if_connected = 0
+    for i in range(1000):
+        G = nx.gnm_random_graph(n, m)
+        if_connected = nx.number_connected_components(G)
+        if if_connected == 1:
+            break
+    if if_connected == 1:
+        return nx.to_dict_of_lists(G)
+    else:
+        return None
```

Comment:
This function without seed does not output the same graph every time. Is this intended?

Reply:
Added random seed argument.